init.lua

```diff
@@ -81,14 +81,19 @@ function SkyRocket:new(options)
 
   local resizer = {
     disabledApps = tableToMap(options.disabledApps or {}),
-    dragging = false,
-    dragType = nil,
     moveStartMouseEvent = buttonNameToEventType(options.moveMouseButton or 'left', 'moveMouseButton'),
     moveModifiers = options.moveModifiers or {'cmd', 'shift'},
     windowCanvas = createResizeCanvas(options.opacity or 0.3),
     resizeStartMouseEvent = buttonNameToEventType(options.resizeMouseButton or 'left', 'resizeMouseButton'),
     resizeModifiers = options.resizeModifiers or {'ctrl', 'shift'},
     targetWindow = nil,
+
+    canDrag = false,
```

Comment:
I added this `canDrag` state to represent when dragging has started in mouse-only mode, but the `mouseActivationThreshold` hasn't yet been crossed.